modules/javafx.graphics/src/main/java/com/sun/glass/events/KeyEvent.java

```diff
@@ -252,11 +252,16 @@ public class KeyEvent {
      * Returns a VK_ code of a key capable of producing the given unicode
      * character with respect to the currently active keyboard layout or
      * VK_UNDEFINED if the character isn't present in the current layout.
+     * The hint is the KeyCode of the key the system is attempting to match.
```

Comment:
is this clear enough or should we say that it's KeyCode.getCode()?

(also Application:746)